src/iop/colorzones.c

```diff
@@ -1112,9 +1112,9 @@ void gui_init(struct dt_iop_module_t *self)
   c->select_by = dt_bauhaus_combobox_new(self);
   dt_bauhaus_widget_set_label(c->select_by, NULL, _("select by"));
   gtk_widget_set_tooltip_text(c->select_by, _("choose selection criterion, will be the abscissa in the graph"));
-  dt_bauhaus_combobox_add(c->select_by, _("hue"));
-  dt_bauhaus_combobox_add(c->select_by, _("saturation"));
-  dt_bauhaus_combobox_add(c->select_by, _("lightness"));
+  dt_bauhaus_combobox_add(c->select_by, C_("select by", "hue"));
```

Comment:
Here the context is more "color", we are talking about hue, saturation or lightness in the "color" context.

Reply:
In a way you're right, of course, however the same "color" context would apply elsewhere as well, while "select by" wouldn't. The preceding label ("select by") was the reason for me wanting these here split, so I could translate "hue", etc. here to go nicely with "select by". (Other inflectional languages might appreciate having this option, too.)

Reply:
Maybe I can't tell for your language, but I'm concerned. Will we have:

   _("select by", "hue")
   _("choose", "hue")
   _("button", "hue")
   ...

Here it seems to me that "hue" is used in the context/theme of color. The context in C_("")  macro is really the context/theme of the term used not an action "select by". Again I don't know much about languages structure, but I want the translation to stay at a minimum level for maintenance.

Reply:
"Hue" is basically always used in the context of color, so this won't really help. What about "colorzones", the module where these occur?

Longer explanation:
As you probably know, there are currently two specified contexts for "hue" in dt:
1) "blendmode"
2) "accel"
... plus the unspecified context which is used everywhere else.
I'd like (or need) to separate three more contexts, in order for the Estonian translation to look pretty:
3) "channelmixer" (the string "gray" in that module has this context already)
4) "color reconstruction precedence" (just "color reconstruction" works just as well)
5) "colorzones" (a.k.a. "select by" or however we agree this to be named :) )
... as the translation would differ slightly in those places.
Of course there would still be the undefined context as well, with its remaining 8 occurrences of "hue".

To lessen the impact of this on existing translations, we could agree that I either do another PR afterwards or amend this one to update existing .po files so they'd retain their current translation of "hue" and other similarly affected strings in these added contexts. (For about 2/3 of the languages I'd have to mark these strings as fuzzy, though, as I don't read all the languages darktable has been translated to, and am therefore unable to decide whether the translation could benefit from the new contexts or not.)

Reply:
Renamed this context to "colorzones" for now - let me know if that's not an improvement :)